src/bankid.js

```diff
@@ -43,14 +43,21 @@ class BankId {
   }
 
   authenticate(endUserIp, personalNumber, requirement) {
+
     if (!endUserIp) {
       throw Error("Missing required argument endUserIp.");
     }
-    return this._call("auth", {
+
+    let options = {
       endUserIp,
-      personalNumber,
       requirement
-    });
+    };
+
+    if(personalNumber) {
+      options.personalNumber = personalNumber;
+    }
```

Comment:
Would you mind adding this change to the sign method as well so we are consistent for how we handle personal number for both authenticate and sign?